actioncable/README.md

```diff
@@ -21,4 +21,4 @@ Bug reports for the Ruby on Rails project can be filed here:
 
 Feature requests should be discussed on the rails-core mailing list here:
 
-* https://groups.google.com/forum/?fromgroups#!forum/rubyonrails-core
+* https://discuss.rubyonrails.org/c/rubyonrails-core/5
```

Comment:
Don't think the extra "5" or "7" are needed?

Reply:
Oh. When I click on the `rubyonrails-core` it redirects to that URL(used in my PR with id at the end) so I used that one. The one you mentioned seems to be working as well. I update it at all the places. Thanks, @vipulnsward